src/windows/scanner.cc

```diff
@@ -245,6 +246,22 @@ drivelist::Scanner::Scan(std::vector<drivelist::disk_s> *const output) {
 
   for (drivelist::disk_s disk : disks) {
     disk.mountpoints = mountpointMap[disk.id];
+
+    // For user friendly purposes, in Windows, the disk name is
+    // simply the list of drive letters, separated by commas.
+    disk.displayName = std::accumulate(
```

Comment:
Uh, I just realized I'm not handling the case where there are no mountpoints :/ I'll fix it in a minute